sources/packages/backend/libs/services/src/constants/system-configurations-constants.ts

```diff
@@ -31,3 +32,5 @@ export const QUEUE_RETRY_DEFAULT_CONFIG = {
    */
   backoff: QUEUE_RETRY_DELAY,
 };
+
+export const IER_SCHEDULER_JOB_ID = QueueNames.IERIntegration;
```

Comment:
Unless it is a blocker, I would suggest not to assign the job id.

Reply:
My understanding is that the `job id` is different from the `job key` which is unique and includes even the corn expression.
What is the impact of having it associated with the job?

Reply:
yes, the job id and key are different.
this job id is internally the id of the repeatable job,
![image](https://user-images.githubusercontent.com/77353155/205332351-7708f53f-cc81-45ba-b67d-199e8e49c78e.png)
Please refer https://github.com/bcgov/SIMS/pull/1582#discussion_r1038261065